startup/BMM/user_ns/base.py

```diff
@@ -12,6 +12,9 @@ def is_re_worker_active():
 
 os.environ['BLUESKY_KAFKA_BOOTSTRAP_SERVERS'] = 'kafka1.nsls2.bnl.gov:9092'
 
+## the intent here is to return $HOME/.profile_collection/startup
+startup_dir = os.path.split(os.path.split(os.path.split(__file__)[0])[0])[0]
```

Comment:
`os.path.dirname` makes is a little easier to read (no indices are needed):
```suggestion
startup_dir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
```

Reply:
agreed